src/api.c

```diff
@@ -124,14 +128,17 @@ API const unsigned int seccomp_api_get(void)
 API int seccomp_api_set(unsigned int level)
 {
 	switch (level) {
-	case 1:
-		sys_set_seccomp_syscall(false);
-		sys_set_seccomp_flag(SECCOMP_FILTER_FLAG_TSYNC, false);
-		break;
+	case 3:
+		sys_set_seccomp_flag(SECCOMP_FILTER_FLAG_LOG, true);
+	/* fallthrough */
```

Comment:
Nitpicky, but since you're probably going to respin for the above comment, please indent this comment to the same level as the code :)

Reply:
This is actually what astyle requires. I originally had it indented like what you're asking for but astyle flagged it as incorrect.

Reply:
Huh, okay, sorry for the noise.